test/test_actuator_server.py

```diff
@@ -60,3 +88,60 @@ def test_server_deserialize():
     assert server.port == 443
     assert server.ssl_certificate == "mycert.pem"
     assert server.ssl_key == "mykey.pem"
+
+
+def test_server_working(actuator_server):
+    thread = actuator_server.run()
+    assert thread is not None
+    time.sleep(3)
+    path = app.routes[0].path.format(actuator_id="test_id")
+    response = requests.get(f"https://127.0.0.1:{HTTPS_PORT}{path}", verify=False)
+    assert response.status_code == 404
```

Comment:
Why the server should give 404 if the test is supposed to check if it's working? It should give back 200 in this case.

Reply:
Because at the moment there are no registered actuators, so the client is looking for a non-existent actuator. The server is correctly working because it is replying with a "404 - not found" error code.